registry.json

```diff
@@ -2430,5 +2430,19 @@
     "telegram": "devdertin",
     "operatorName": "Guillermo Céspedes",
     "github": "terostakepool"
-  }
+  },
+  {
+ "ticker": "ALTZ",                                                       
+ "poolId": "46ebbc0d27d9169cae12d12dcf8f6cacbcf27937c67ae4350abbcaba",  
+ "social": {              
+   "website":"https://altzpool.com", 
+   "twitter": "@AltzPool",
+   "facebook": " ",  
+   "instagram": " ",
+   "youtube": " "
+ },
+ "telegram": "ALTZ1",                                             
```

Comment:
Please provide us with a Telegram username.  Groups or channel are not allowed in this field.  We need a username to leverage Telegram voting features and let you voice your opinion in the CSPA decision making.

Reply:
do I have to re-register again? @altzpool is the username.